spaceship/lib/spaceship/connect_api/api_client.rb

```diff
@@ -221,6 +222,14 @@ def handle_error(response)
         end
       end
 
+      def response_body_as_json(body)
+        if body.kind_of?(Hash)
+          return body
+        end
+
+        return JSON.parse(body)
```

Comment:
We might want to rescue an error here if it can't parse and display something _somewhat_ user friendly 🤔 Thoughts?

Reply:
I have changed this part of the code, but I am doing the same as in other places.